### ISSUE TITLE
after downloading an add-on using the nvda store and hitting the close button nda does not provide a fidback that the add-ons are being installed.

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14980]
### Steps to reproduce:
download an nvda add-on
in my case I used both tabs:
Updateable add-ons
It is
Available add-ons
Now press the close button

### Actual behavior:
the nvda is silent considering that it crashed.

### Expected behavior:
a window should be displayed and verbalized by nvda asking the user to wait while the add-ons are being installed
This is important when you want to update or install many add-ons.
Also relevant when running nvda portable on a slow drive.

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-28355,02199b76

#### Windows version:
10 19045.2965
#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by fernando-jose-silva** [comment]
thank you very much, I didn't have this problem anymore, I don't know what happened here, we can close.
